rts/Lua/LuaSyncedRead.cpp

```diff
@@ -1686,12 +1686,12 @@ int LuaSyncedRead::GetTeamList(lua_State* L)
 }
 
 
-/***
+/*** Get player IDs
  *
  * @function Spring.GetPlayerList
- * @param teamID integer? (Default: `-1`) to filter by when >= 0
- * @param active boolean? (Default: `false`) whether to filter only active teams
- * @return number[]? list of playerIDs
+ * @param teamID integer? (Default: `-1`) when >= 0 filters by teamID and filters non-spectator players if `active` is true
```

Comment:
Doesn't the "and filters non-spectator players if `active` is true" belong on the next line, for the `active` var? If `active` is ignored unless teamID is set then that may also be a bug.